policyengine_us/parameters/gov/aca/la_county_rating_area.yaml

```
@@ -0,0 +1,50 @@
+description: Rating areas for Los Angeles County based on 3-digit zip codes
```

Or whatever the standard name is for the rating areas
```suggestion
description: Los Angeles County assigns three-digit ZIP codes to these Affordable Care Act rating areas.
```